tests/Unit/PointwiseFunctions/Elasticity/ConstitutiveRelations/Test_IsotropicHomogeneous.cpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp"
+#include "tests/Unit/Pypp/CheckWithRandomValues.hpp"
+#include "tests/Unit/Pypp/SetupLocalPythonEnvironment.hpp"
+#include "tests/Unit/TestCreation.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+test_isotropic_homogeneous(const double incompressibility,
+                           const double rigidity) {
+  const Elasticity::ConstitutiveRelations::IsotropicHomogeneous<3> relation{
+      incompressibility, rigidity};
+  test_serialization(relation);
+
```

Comment:
Depending on how you'll use `ConstitutiveRelations`, you might also want to add a `test_move` and a `test_from_options` as we do in hydro analytic data.

Reply:
good point, added those tests